File: python/Maximum.py
Numbers = []
Size = int(input("Among how many numbers do you want to find the maximum?"))
Numbers.append(int(input("Enter the first number:")))
for Counter in range(1, Size):
  Numbers.append(int(input("Enter the next number:")))
Max = Numbers[0]
for Counter in range(1, Size):
  if(Max < Numbers[Counter]):
    Max = Numbers[Counter]
print('The maximum among {0} numbers is {1}.' .format(Size, Max))

File: c/Maximum.c
#include <stdio.h>

int main(void)
{
  int Numbers[20], Counter, Maximum, Size;
  printf("Among how many numbers do you want to find the maximum?");
  scanf("%d", &Size);
  printf("Enter the first number:");
  scanf("%d", &Numbers[0]);
  for(Counter = 1; Counter < Size; Counter++)
  {
     printf("Enter the next number:");
     scanf("%d", &Numbers[Counter]);
  }
  Maximum = Numbers[0];
  for(Counter = 1; Counter < Size; Counter++)
  {
    if(Maximum < Numbers[Counter])
    {
      Maximum = Numbers[Counter];
    }
  }
  printf("The maximum of %d numbers is %d.\n", Size, Maximum);
  return 0;
}